src/atoms/TextInput/AnimatedLabel.native.js

```diff
@@ -0,0 +1,212 @@
+import React, { useContext, useRef, useEffect } from 'react';
+import { Animated, Platform } from 'react-native';
+import styled, { ThemeContext } from 'styled-components/native';
+import PropTypes from 'prop-types';
+import { getColor } from '../../_helpers/theme';
+import View from '../View';
+
+const IS_ANDROID = Platform.OS === 'android';
+
+const ANDROID_OUTLINED_INITIAL_TOP_DIVISOR = 2.4;
+const IOS_OUTLINED_INITIAL_TOP_DIVISOR = 1.9;
+
+const ANDROID_FILLED_INITIAL_TOP_DIVISOR = 1.15;
+const IOS_FILLED_INITIAL_TOP_DIVISOR = 1.3;
+
+const styles = {
+  container: {
+    height({ variant }) {
+      if (variant === 'outlined') {
+        return IS_ANDROID ? '2px' : '10px';
+      } else {
+        return IS_ANDROID ? '20px' : '20px';
+      }
+    },
+  },
+  text: {
+    color({ theme }) {
+      return getColor(theme, 'shade.980');
+    },
+    fontSize({ theme }) {
+      return theme.fonts.size.xsmall;
+    },
+    lineHeight({ theme }) {
+      return theme.fonts.lineHeight.small;
+    },
+    fontFamily({ theme }) {
+      return theme.fonts.family.lato.regular;
+    },
+  },
+};
+
+const Container = styled(View)`
```

Comment:
We need to create a Size enhancer component for this

Reply:
Done. Using it throughout the PR now.